modules/core/02-client/migrations/v7/solomachine.go

```diff
@@ -0,0 +1,263 @@
+package v7
```

Comment:
copied from `legacy/v100/solomachine.go`

Needed in order to unmarshal the existing types which no longer register solomachine v2 on the codec. The SDK codec will prematurely try to lookup type registration against the interface (even if the cdc isn't supposed to unpack the interface).